WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -403,6 +410,21 @@ public void onClick(View v) {
                         mAccountStore.getAccount().getUserName());
             }
         });
+
+        mQuickStartContainer.findViewById(R.id.row_quick_start).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (mQuickStartDot.getVisibility() == View.VISIBLE) {
+                    AppPrefs.setQuickStartActive(false);
+                    mQuickStartContainer.setVisibility(View.GONE);
+                }
+
+                ActivityLauncher.viewQuickStartForResult(getActivity());
+            }
+        });
+
+        mToolbar = rootView.findViewById(R.id.toolbar_main);
+        mToolbar.setTitle(mToolbarTitle);
```

Comment:
These two `mToolbar` statements are [duplicated in `onCreateView`](https://github.com/wordpress-mobile/WordPress-Android/blob/20730178d7e9eb565de4118d75d55a1db345b264/WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java#L236-L237).  I think they may have been accidentally copied/pasted and can be removed.